themes/default/content/blog/finops-with-pulumi/index.md

```diff
@@ -0,0 +1,126 @@
+
+## Pulumi Features in Context
+
+### Multi Language Components
+
+Multi-Language components allow you to create repeatable infrastructure patterns, modular abstractions of resources and/or groups of resources that meet your desired configuration standards.  Packaging up resources within and across clouds, implementing standard deployment best practices and making those components available to developers reduces the cognitive load that they have when building new applications from those pre-assembled, ready-to-run components.
+
+Use these guardrails to enforce Tag Policies, for example that every tag for “Production” is actually “Production” and not “Prod” or “production” (yes, case sensitivity matters).  Or you may ensure teams only deploy resources such as VMs to regions where you have allocated Reserved Instances or other financial benefits in your application architecture. With MLCs end users have access to your component and not to the underlying infrastructure resources, guaranteeing repeatabality, however, in order to realize the benefits of MLCs, your Developers must be empowered to leverage them, but limited from using other methods.
+
+## CrossGuard
+
+CrossGuard Policy as Code allows an Organization to define global rules for how resources may be provisioned and prevents resources from being provisioned by Pulumi in any other configuration.  Pulumi Stacks that do not meet the Organization’s financial, security, data protection or other pre-defined standards are prevented from provisioning any resources and the developer is instructed to correct their desired configuration or program in accordance with the Organization-wide policies.
+
+Policy as Code ensures that any Pulumi code written and run meets an Organization’s stated policies. It is the ultimate preventative weapon to prevent Financial, Security, or other placement and provisioning decisions that may lead to exposure or overrun.
+
+Go here for [examples of FinOps policies](https://github.com/pulumi/examples/tree/master/policy-packs/aws-ts-finops) to get you started!
+
+## Automation API
+
+Automation API allows you to create an HTTP endpoint for a Pulumi program.  Organizations have used this to control easy on/off switches for applications such as provisioning/deprovisioning single tenant SaaS architectures, short lived environments and complex testing and blue/green deployments.  Because of the encapsulation of a Pulumi program and the simplification this provides, this endpoint may be directly exposed to those responsible for making the financial decisions in the moment, such as a Sales Engineer who must turn on a new POC environment for testing who then must be sure to shut that off when completed.  Oftentimes these decisions are automated in process and pipelines.
```

Comment:
This is obviously not the only thing one can do with Automation API. You may want to consider adding "Among other things" or "Among the various things you can do with Automation API, you can create an..." or something like that.